components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/connector/AdminForcedPasswordResetConfigImpl.java

```diff
@@ -135,4 +147,22 @@ public Properties getDefaultPropertyValues(String tenantDomain) throws IdentityG
         return null;
     }
 
+    private Property getPropertyObject(String type) {
+        Property property = new Property();
```

Comment:
Let's add a new line after the method signature. You can resolve this using by formatting the code.

Lets fix in all other places

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767